src/components/node-list/index.js

```diff
@@ -86,6 +90,9 @@ const NodeListProvider = ({
       onToggleTagActive(item.id, true);
     } else if (isModularPipelineType(item.type)) {
       onToggleModularPipelineActive(item.id, true);
+    } else if (isElementType(item.type) && item.id === 'parameters') {
```

Comment:
as mentioned above, `isElementType` was not declared in `node-list-items.js` - the `isElementType` check does not seem to be needed?